src/widgets/_emojipicker.scss

```
@@ -0,0 +1,58 @@
+.emoji-picker {
+    entry {
+        margin: rem(12px);
+        margin-bottom: 0;
+    }
+
+    // Emoji are labels too, so we have to be specific
+    box.vertical > label {
+        font-weight: 700;
+        //Intentionally off-by-one
+        margin-left: rem(7px);
+        margin-right: rem(7px);
+        opacity: 0.8;
+    }
+
+    .emoji {
+        min-height: 48px;
+        min-width: 48px;
```

This seems pretty spaced out, reducing the amount of emoji that can be seen without scrolling. What about 36px here?

```suggestion
        min-height: 36px;
        min-width: 36px;
```